### PR TITLE
 OP-12962-Master script changes for aanalytics for dropping constrina…

### PR DESCRIPTION
**Pull request by srikanthopsmx**
…s for delete application issue
https://devopsmx.atlassian.net/browse/OP-12962

### CODE REVIEW

File: scripts/oes-data-migration-scripts/migration_v3.11.x_to_v3.12.py [hunk 1]
```diff
@@ -35,15 +35,32 @@ def perform_migration():
                 migrate_user_group_permission(user_group_permissions)
             if step == 13:
                 platform_conn.commit()
+            if step == 14:
+            	updateautopilotconstraints()
+        	    autopilot_conn.commit()
```
Comment (Pranav-b-7) [inline scripts/oes-data-migration-scripts/migration_v3.11.x_to_v3.12.py:40]: Why is there an extra intendation here ?

File: scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.11.x_to_v3.12.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 2. You will find a file named : migration_v3.11.x_to_v3.12.py
 3. Run the below command to execute the script.
 
-        python3 <scriptfilename> <platform_db-name> <platform_host> <oes-db-name> <oes-db-host> <db-port>
-
-    ex: python3 migration_v3.11_to_v3.12.py platformdb oes-db oesdb oes-db 5432
+	python3 <scriptfilename> <platform_db-name> <platform_host> <oes-db-name> <oes-db-host> <autopilot_db> <autopilot_host> <db-port>
+	
+    ex: python3 migration_v3.11_to_v3.12.py platformdb oes-db oesdb oes-db  opsmx localhost 5432
```
Comment (Pranav-b-7) [inline scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.11.x_to_v3.12.txt:7]: instead of localhost . give oes-db

**Review by Pranav-b-7 — CHANGES_REQUESTED** [review]

File: scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.11.x_to_v3.12.txt (hunk 2 above)
Comment (srikanthopsmx) [reply scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.11.x_to_v3.12.txt:7]: k

File: scripts/oes-data-migration-scripts/migration_v3.11.x_to_v3.12.py (hunk 1 above)
Comment (srikanthopsmx) [reply scripts/oes-data-migration-scripts/migration_v3.11.x_to_v3.12.py:40]: will check

**Review by Pranav-b-7 — APPROVED** [review]

**Pull request merged**